src/js/site/main.js

```diff
@@ -402,3 +402,47 @@ $(document).ready(function () {
 
 // Currently this function is only used for the insights table
 $(document).ready(highlightURLHash());
+
+// update date shown to be X ago
+$(document).ready(function () {
+  function ago(date) {
+    function render(n, unit) {
+      return n + ' ' + unit + (n == 1 ? '' : 'S') + ' AGO';
```

Comment:
`===` ?

Reply:
I saw most of our code uses double = so I didn't use triple =, I can update

Reply:
i think that might be for other reasons.

Reply:
in this case triple equals makes sense.

Reply:
Sounds good!